internal/commands/certificates.go

```diff
@@ -1,22 +1,16 @@
 package commands
 
 import (
-	"crypto/ecdsa"
-	"crypto/ed25519"
 	"crypto/elliptic"
-	"crypto/rand"
-	"crypto/rsa"
-	"crypto/x509"
-	"crypto/x509/pkix"
-	"encoding/pem"
 	"fmt"
+	"io/ioutil"
```

Comment:
The stdlib package io/ioutil is discouraged for new code from 1.16 onwards., there should be equivalent methods in the io or os packages.